pyClient/test_commands/scenario.py

```diff
@@ -396,3 +412,199 @@ def charlie_double_withdraw(
         4000000,
         zksnark
     )
+
+
+def charlie_corrupt_bob_deposit(
+        prover_client: ProverClient,
+        mixer_instance: Any,
+        mk_root: str,
+        bob_eth_address: str,
+        charlie_eth_address: str,
+        keystore: mock.Keystore,
+        mk_tree_depth: int,
+        zksnark: str) -> contracts.MixResult:
+    """
+    Charlie tries to break transaction malleability and corrupt the coins
+    bob is sending in a transaction
+    She does so by intercepting bob's transaction and either:
+    - case 1: replacing the ciphertexts (or pk_sender) to garbage
```

Comment:
`to garbage` -> replacing the ciphertexts by garbage/arbitrary data

Reply:
Corrected